.bazelrc

```diff
@@ -10,6 +10,7 @@ build --action_env=CONTROLLER_IMAGE=quay.io/konveyor/forklift-controller:latest
 build --action_env=MUST_GATHER_IMAGE=quay.io/konveyor/forklift-must-gather:latest
 build --action_env=MUST_GATHER_API_IMAGE=quay.io/konveyor/forklift-must-gather-api:latest
 build --action_env=UI_IMAGE=quay.io/konveyor/forklift-ui:latest
+build --action_env=UI_PLUGIN_IMAGE=quay.io/kubevirt-ui/forklift-console-plugin:latest
```

Comment:
note to self, make sure `latest` is not broken ... :-)

Reply:
well everybody should make that note on all projects + the latest is (or should be) just for devs :D